deploy/chart/crds/0000_50_olm_00-catalogsources.yaml

```diff
@@ -1,190 +1,193 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: catalogsources.operators.coreos.com
   annotations:
-    displayName: CatalogSource
-    description: A source configured to find packages and updates.
+    controller-gen.kubebuilder.io/version: v0.2.8
+  creationTimestamp: null
+  name: catalogsources.operators.coreos.com
 spec:
   group: operators.coreos.com
-  version: v1alpha1
-  versions:
-  - name: v1alpha1
-    served: true
-    storage: true
-  preserveUnknownFields: false
-  scope: Namespaced
   names:
-    plural: catalogsources
-    singular: catalogsource
+    categories:
+    - olm
```

Comment:
What does `-olm` do in this case?

Reply:
You can “kubectl get olm” to get all olm related resources

Reply:
Great!